featuretools/entityset/entityset.py

```diff
@@ -925,8 +925,11 @@ def plot(self, to_file=None):
         for entity in self.entities:
             variables_string = '\l'.join([var.id + ' : ' + var.type_string  # noqa: W605
                                           for var in entity.variables])
-            nrows = entity.shape[0]
-            label = '{%s (%d row%s)|%s\l}' % (entity.id, nrows, 's' * (nrows > 1), variables_string)  # noqa: W605
+            if isinstance(entity.df, dd.DataFrame):  # entity is a dask entity
+                label = '{%s |%s\l}' % (entity.id, variables_string)  # noqa: W605
+            else:
+                nrows = entity.shape[0]
+                label = '{%s (%d row%s)|%s\l}' % (entity.id, nrows, 's' * (nrows > 1), variables_string)  # noqa: W605
```

Comment:
@rwedge if this looks good I will start working on the broken test units to finish it up.

Reply:
Looks good!